ext/json/tests/exceptions.phpt

```diff
@@ -0,0 +1,130 @@
+--TEST--
```

Comment:
Please could you add a separate tests for json_decode and json_encode. It's much more useful when doing changes in the extension to immediately see what parts are failing. Probably mainly useful for me though... :)